classifier/__init__.py

```diff
@@ -973,7 +978,10 @@ def scrub_identifier(cls, identifier):
             identifier = int(identifier)
         except ValueError:
             pass
-        return identifier
+
+        # For our purposes, Dewey Decimal numbers are identifiers
+        # without names.
+        return identifier, None
```

Comment:
Without this, the identifier (which is an integer here) is used as the name, and `scrub_identifier` choked trying to put it into a `Lowercased`.